eng/code-quality-reports/src/main/resources/checkstyle/checkstyle.xml

```diff
@@ -278,7 +278,7 @@ page at http://checkstyle.sourceforge.net/config.html -->
     <!-- Javadoc checks                                   -->
     <!-- See http://checkstyle.sf.net/config_javadoc.html -->
     <module name="JavadocMethod">
-      <property name="scope" value="protected"/>
+      <property name="accessModifiers" value="public, protected"/>
```

Comment:
nit: Should there be a space between `public,` and `protected`?

Reply:
We have a space for other configs and it appears to work still. xD